README.md

```diff
@@ -164,11 +185,11 @@ Students are responsible for making sure their assignments are *reproducible*, a
 
 In short: whatever grade the autograder returns after 5 minutes (assuming the teaching team did not make an error) is the grade that will be assigned.
 
-### Re-grading
+<!--### Re-grading
 
 Students get a free 1% at the end of the course. If you want questions regraded, you can fill out a form at the end of the term documenting in detail what you want regraded. Regrading is only offered if the teaching team makes a mistake. If the total sum of value of possible points via regrading is greater than 1%, you can submit (please do so before the last day of class); otherwise the regrade request is rejected. Note: If you submit a regrade request, you may get less than 1% via regrade.
 
-Note if you received 0 on an assignment when you shouldn't have, please contact the course coordinator as soon as possible rather than waiting until the end of the term.
+Note if you received 0 on an assignment when you shouldn't have, please contact the course coordinator as soon as possible rather than waiting until the end of the term. -->
```

Comment:
@XiziPeng  Is it correct that we are removing this section?

Reply:
@joelostblom We decided to keep it!